src/Domain/Creators/SphereTimeDependentMaps.hpp

```diff
@@ -223,28 +277,32 @@ struct TimeDependentMapOptions {
       bool include_distorted_map) const;
 
   /*!
-   * \brief This will construct the map from `Frame::Grid` to `Frame::Inertial`.
+   * \brief This will construct the map from `Frame::Grid` to
+   * `Frame::Inertial`.
    *
-   * If the argument `include_distorted_map` is true, then this map will have a
-   * `Shape` map (with a size function of time). If it is false, then there will
-   * only be an identity map.
+   * If the argument `include_distorted_map` is true, then this map will
+   * have a `Shape` map (with a size function of time). If it is false, then
+   * there will only be an identity map.
    */
   MapType<Frame::Grid, Frame::Inertial> grid_to_inertial_map(
       bool include_distorted_map, bool use_rigid_translation) const;
 
   inline static const std::string size_name{"Size"};
   inline static const std::string shape_name{"Shape"};
+  inline static const std::string rotation_name{"Rotation"};
+  inline static const std::string expansion_name{"Expansion"};
+  inline static const std::string expansion_outer_name{"ExpansionOuter"};
```

Comment:
Please use `ExpansionOuterBoundary` for the outer name so that it's consistent with the BBH.

Reply:
Renamed